File: apis/TradeOffAnalytics/python/TradeoffAnalyticsClient.py
import json
import os
from os.path import join, dirname
from watson_developer_cloud import TradeoffAnalyticsV1

tradeoff_analytics = TradeoffAnalyticsV1(
  username='cc35d4e3-a15b-4052-958d-fe0e4f40f9ca',
  password='xX62zUw36Gd4')

with open(os.path.join(os.path.dirname(__file__), '../resources/problem.json')) as problem_json:
  dilemma = tradeoff_analytics.dilemmas(json.load(problem_json),
                                        generate_visualization=False)

print(json.dumps(dilemma, indent=2))

File: apis/TradeOffAnalytics/node/TradeoffanalyticsClient.js
'use strict';

var TradeoffAnalyticsV1 = require('watson-developer-cloud/tradeoff-analytics/v1');
var tradeoff_analytics = new TradeoffAnalyticsV1({
  username: 'cc35d4e3-a15b-4052-958d-fe0e4f40f9ca',
  password: 'xX62zUw36Gd4'
});

var params = require('../resources/problem.json');

tradeoff_analytics.dilemmas(params, function(error, resolution) {
  if (error)
    console.log('error:', error)
  else
    console.log(JSON.stringify(resolution, null, 2));
});
